larpandoracontent/LArHelpers/LArMCParticleHelper.cc

```diff
@@ -239,6 +239,42 @@ const MCParticle *LArMCParticleHelper::GetParentMCParticle(const MCParticle *con
 
 //------------------------------------------------------------------------------------------------------------------------------------------
 
+void LArMCParticleHelper::GetAllDescendentMCParticles(const pandora::MCParticle *const pMCParticle,
```

Comment:
Double check to see if there's anything similar already (maybe annoyingly subtly different). These new functions otherwise seem well-motivated and well-named.

Reply:
As far as I can see, there is nothing quite like the descendent and ancestor functions